ui/plugins/pluginpanel.html

```diff
@@ -116,6 +107,24 @@
             </a>
           </li>
         </ul>
+        <ul class="nav navbar-nav navbar-right">
+          <li>
+            <button
+               class="dropdown-toggle icon-blue"
+               data-toggle="dropdown">
+              <span class="fa fa-ellipsis-h" aria-hidden="true"></span>
+            </button>
+            <ul class="dropdown-menu">
+              <li class="{{ canBackfill() ? '' : 'disabled' }}">
+                <a ng-attr-title="{{ backfillButtonTitle() }}"
+                   ng-click="!canBackfill() || backfillJob()">Backfill</a>
+              </li>
+              <li ng-repeat="taskclusterOption in taskclusterOptions">
+                <a href="{{taskclusterOption.url}}">{{taskclusterOption.title}}</a>
```

Comment:
Should this be a `target="_blank"` so either **One Click Loaner** or **Inspect Task** open a new browser tab? I defer to you guys if this is intended to send the user away from Treeherder, which it currently does.